crates/containerd-shim-wasm-test/src/modules/has_default_devices.rs

```diff
@@ -0,0 +1,21 @@
+use std::path::Path;
```

Comment:
Why are some of the modules "wat" and some of are "rs"? For these Rust source files, do you have a step to compile them to wasm modules for testing?

Reply:
the crate's buildscript compiles the files based on the extension